internal/events/aggregator.go

```diff
@@ -387,10 +439,15 @@ func (ag *aggregator) attemptMessageDispatch(ctx context.Context, msg *fftypes.M
 
 		// Generate the appropriate event
 		event := fftypes.NewEvent(eventType, msg.Header.Namespace, msg.Header.ID, tx)
+		event.Correlator = msg.Header.CID
+		if customCorrelator != nil {
+			// Definition handlers can set a custom event correlator (such as a token pool ID)
+			event.Correlator = customCorrelator
+		}
 		if err = ag.database.InsertEvent(ctx, event); err != nil {
 			return err
 		}
-		log.L(ctx).Infof("Emitting %s %s for message %s:%s", eventType, event.ID, msg.Header.Namespace, msg.Header.ID)
+		log.L(ctx).Infof("Emitting %s %s for message %s:%s (c=%v)", eventType, event.ID, msg.Header.Namespace, msg.Header.ID, msg.Header.CID)
```

Comment:
probably should spell out `correlator=` or at least `cid=`, because I'll definitely forget what this log means

Reply:
and should we actually log `event.Correlator` here, which may or may not be the same as `msg.Header.CID`?

Reply:
Looks like this one still fell through the cracks?